### DISCUSSION TITLE
Documentation about onDetachedFromWindow

### DISCUSSION THREAD

**Discussion by ku8ar** [software-mansion/react-native-screens#976] [Ideas]
Regarding this:
https://github.com/DylanVann/react-native-fast-image/pull/773
https://github.com/react-native-svg/react-native-svg/pull/1542
https://github.com/software-mansion/react-native-screens/issues/773

From what I understand, for the libraries to work properly with react-native-screens, we need to change their lifecycle.
Recently, I found a new similar bug here: https://github.com/react-native-masked-view/masked-view/issues/114

And I could try to fix it, but I don't know how. Fixes in fast-image and svg have many commits and author himself wrote "that he is not sure that it work".

Therefore, I am asking for a nice and simple example of how component that supports react-native-screens should look like.

**Comment by WoLewicki** [comment]
The double detach problem should be fixed in `3.4.0` and it was a bug on our side to call `onDetachedFromWindow` twice.
